vaporetto/src/sentence.rs

```diff
@@ -1089,6 +1095,13 @@ impl<'a, 'b> Sentence<'a, 'b> {
         &mut self.tags
     }
 
+    /// Returns a slice of tag scores.
+    #[allow(clippy::type_complexity)]
+    #[inline]
```

Comment:
```suggestion
    #[doc(hidden)]
    #[inline]
```
If we do not introduce an explanatory struct for the returned value, I think it would be better to hide it from the documentation. It would be difficult for anyone other than contributors to use this function.